services/redis.rst

```diff
@@ -43,7 +43,7 @@ To use it in your application, add it to ``.symfony/services.yaml``:
 
 .. code-block:: yaml
 
-    redisdata:
+    myredis:
```

Comment:
I would have prefered to keep redisdata everywhere.

Reply:
Are you sure? I'm ok with that but we use `myservice` in most of our examples.